tests/models.py

```diff
@@ -1352,3 +1353,89 @@ def test_operator_clear(self):
         self.assertEqual(ti2.try_number, 1)
         # try_number (0) + retries(1)
         self.assertEqual(ti2.max_tries, 1)
+
+    def test_xcom_disable_pickle_type(self):
```

Comment:
Hmm I just meant the above thing that applying the schema change shouldn't delete the values in the DB in case they need to be recovered/are important.